src/datatip.jl

```diff
@@ -8,21 +8,59 @@ handle("datatip") do data
   @destruct [
     word,
     mod || "Main",
-    path || nothing,
+    path || "",
+    column || 1,
     row || 1,
-    column || 1
+    startRow || 0,
+    context || ""
   ] = data
 
+  datatip(word, mod, path, column, row, startRow, context)
+end
+
+function datatip(word, mod, path, column = 1, row = 1, startRow = 0, context = "")
   if isdebugging() && (datatip = JunoDebugger.datatip(word, path, row, column)) !== nothing
     return Dict(:error => false, :strings => datatip)
   end
 
+  let localdatatip = localdatatip(word, column, row, startRow, context)
+    localdatatip !== nothing && return Dict(:error => false, :line => localdatatip)
```

Comment:
So the datatip is going to show the line where `word` is defined. Obviously it's not complete -- the whole binding won't be covered for some cases like
```jl
val = begin
  ...
end
```
(with the current implementation the datatip would only show `val = begin`)

As an alternative, I tried to implement a function that shows the whole binding from `CSTParser.Binding`, but it didn't work well ...

Reply:
addressed in d3b5c5a